web-common/src/features/canvas/stores/syncCanvasState.ts

```diff
@@ -0,0 +1,15 @@
+import type { StateManagers } from "@rilldata/web-common/features/canvas/state-managers/state-managers";
+import { canvasEntityStore } from "@rilldata/web-common/features/canvas/stores/canvas-stores";
+import { get } from "svelte/store";
+
+export function createCanvasStateSync(ctx: StateManagers) {
+  const canvasName = get(ctx.canvasName);
+  if (canvasName in get(canvasEntityStore).entities) {
+    // TODO: Add sync method if required
+    return { isFetching: false, error: false };
```

Comment:
When we get around to implementing this state sync, could we move it into the `CanvasEntity` class itself? Then we can encapsulate/colocate all logic around state there. If so, we could probably remove this file altogether (though, just need to find a better place for the `canvasEntities.addCanvas(canvasName)` call).